fortnite_api/enums.py

```diff
@@ -47,7 +49,133 @@
 )
 
 
-class KeyFormat(enum.Enum):
+E = TypeVar('E', bound='Enum')
+
+
+def _create_value_cls(name: str, comparable: bool):
+    # All the type ignores here are due to the type checker being unable to recognise
+    # Runtime type creation without exploding.
+    cls = namedtuple('_EnumValue_' + name, 'name value')
+    cls.__repr__ = lambda self: f'<{name}.{self.name}: {self.value!r}>'  # type: ignore
+    cls.__str__ = lambda self: f'{name}.{self.name}'  # type: ignore
+    if comparable:
+        cls.__le__ = lambda self, other: isinstance(other, self.__class__) and self.value <= other.value  # type: ignore
+        cls.__ge__ = lambda self, other: isinstance(other, self.__class__) and self.value >= other.value  # type: ignore
+        cls.__lt__ = lambda self, other: isinstance(other, self.__class__) and self.value < other.value  # type: ignore
+        cls.__gt__ = lambda self, other: isinstance(other, self.__class__) and self.value > other.value  # type: ignore
+    return cls
+
+
+def _is_descriptor(obj):
```

Comment:
Missing annotations here,

```suggestion
def _is_descriptor(obj: Type[object]) -> bool:
```

As a side note, `typing.TypeIs` may fit well here alongside an internally defined `Protocol` in the `TYPE_CHECKING` block,
```python
if TYPE_CHECKING:
    class _SetterObj(typing.Protocol):
        # Defined methods
        def __get__(...): ...

        # ...

def predicate(obj: Type[object]) -> TypeGuard[_SetterObject]: ...
```
What do you think?